trustmanager/keyfilestore.go

```diff
@@ -0,0 +1,56 @@
+package trustmanager
+
+import "errors"
+
+const (
+	keyExtension = "key"
+)
+
+// KeyFileStore persists and manages private keys on disk
+type KeyFileStore struct {
+	FileStore
+}
+
+// NewKeyFileStore returns a new KeyFileStore creating a private directory to
+// hold the keys.
+func NewKeyFileStore(baseDir string) (EncryptedFileStore, error) {
```

Comment:
As a general rule, NewXxx functions should return type Xxx, not the interface it implements. In the case that the interface lives in a different package it saves a dependency. It also takes greater advantage of Go's pseudo-duck typing when dealing with interfaces.